src/docs/devices/Brilliant-Smart-Plug/index.md

```diff
@@ -26,10 +26,11 @@ substitutions:
 esphome:
   name: $devicename
   friendly_name: $friendlyname
-  platform: ESP8266
-  board: esp01_1m
   board_flash_mode: dout
```

Comment:
This PR should also include moving `board_flash_mode` from the `esphome` block to `board_flash_mode` inside the new `esp8266` block